model/metrics/system-metrics.yaml

```
@@ -61,6 +61,15 @@ groups:
       - ref: system.cpu.state
       - ref: system.cpu.logical_number
 
+  - id: metric.system.cpu.frequency
+    type: metric
+    metric_name: system.cpu.frequency
+    brief: "Reports the current frequency of the CPU processor in Hz"
```

CPU's `P` is `processor`:

```suggestion
    brief: "Reports the current frequency of the CPU in Hz"
```